twisted/conch/ssh/keys.py

```diff
@@ -4,6 +4,9 @@
 
 """
 Handling of RSA and DSA keys.
+
+@ivar curveTable: a look up table for translating NIST standard curve names to Cryptograpy instances.
```

Comment:
This should be `@var` and not `@ivar`, since modules aren't classes, they don't have instances - `@ivar` stands for "instance variable".

The reference documentation for this sort of things is http://epydoc.sourceforge.net/epytext.html